flickr_download/flick_download.py

```diff
@@ -113,14 +113,21 @@ def download_set(set_id, get_filename, size_label=None):
 
     for photo in photos:
         fname = get_full_path(dirname, get_filename(pset, photo, suffix))
+        if 'video' in photo.getInfo():
+            fname = fname[:-3] + 'mp4'
         if os.path.exists(fname):
             # TODO: Ideally we should check for file size / md5 here
             # to handle failed downloads.
             print('Skipping {0}, as it exists already'.format(fname))
             continue
 
         print('Saving: {0}'.format(fname))
-        photo.save(fname, size_label)
+
+        if ('video' in photo.getInfo()):
+            photo.save(fname, 'HD MP4')
+        else:
+            photo.save(fname, size_label)
+
```

Comment:
(there's one too many empty lines here which breaks the pep8 rules, but this block should be reverted when we move all the logic up above)